socorro/external/boto/crashstorage.py

```diff
@@ -292,6 +292,20 @@ class TelemetryBotoS3CrashStorage(BotoS3CrashStorage):
         reference_value_from='resource.elasticsearch',
     )
 
+    required_config.add_option(
+        'keybuilder_class',
```

Comment:
Why not instead override `__init__` and change the config there explicitly before calling `super().__init__()` which instantiates the connection context?